invenio_records_rest/serializers.py

```diff
@@ -26,45 +26,132 @@
 
 from __future__ import absolute_import, print_function
 
+import pytz
 from flask import jsonify, url_for
 
 
-def record_self_link(pid, record, **kwargs):
+def record_self_link(pid_value, pid_type, record, **kwargs):
     """Create self link to a given record.
 
     :Parameters:
+        - `pid_value` (str): pid value.
+        - `pid_type` (str): type of the pid.
         - `record` (Record): record to which the generated link will point.
         - `**kwargs`: additional parameters given to flask.url_for.
     :Returns: link pointing to the given record.
     :Returns Type: str
     """
     return url_for(
-        "invenio_records_rest.{0}_item".format(pid.pid_type),
-        pid_value=pid.pid_value, **kwargs)
+        'invenio_records_rest.{0}_item'.format(pid_type),
+        pid_value=pid_value, **kwargs)
 
 
-def record_to_json_serializer(pid, record, code=200, headers=None):
+def record_to_json_serializer(pid, record, code=200,
+                              headers=None):
     """Build a json flask response using the given record.
 
+    :Parameters:
+        - `pid`: record's pid.
+        - `record` (Record): record to which the generated link will point.
+        - `code` (int): http code of the response.
+        - `header`: header to extend for the response.
     :Returns: A flask response with json data.
     :Returns Type: :py:class:`flask.Response`
     """
     # FIXME: use a formatter instead once it is implemented
-    self_link = record_self_link(pid, record, _external=True)
-    response = jsonify({
+    self_link = record_self_link(pid.pid_value, pid.pid_type, record,
+                                 _external=True)
+    formatted_record = {
         'id': pid.pid_value,
         'metadata': record,
         'links': {
             'self': self_link
         },
         # FIXME: ISO8601 encoded timestamps in UTC
-        'created': record.model.created,
-        'updated': record.model.updated,
-        'revision': record.model.version_id,
-    })
+        'created': pytz.utc.localize(record.created).isoformat(),
+        'updated': pytz.utc.localize(record.updated).isoformat(),
+        'revision': record.revision_id,
+    }
+    response = jsonify(formatted_record)
     response.status_code = code
     if headers is not None:
         response.headers.extend(headers)
     response.headers['location'] = self_link
     response.set_etag(str(record.model.version_id))
     return response
+
+
+def search_to_json_serializer_factory(hit_formatter,
+                                      aggregations_formatter=None):
+    """Create a search result to flask response serializers.
+
+    :Parameters:
```

Comment:
My bad, I thought that it was a valid alternative syntax but it's in fact the *consolidated fields* syntax of Epydoc.
I will fix that right away.